packages/cactus-plugin-ledger-connector-besu/src/test/typescript/benchmark/run-plugin-ledger-connector-besu-benchmark.ts

```diff
@@ -0,0 +1,239 @@
+import path from "path";
+import { EOL } from "os";
+import * as Benchmark from "benchmark";
+
+import test, { Test } from "tape-promise/tape";
```

Comment:
Please use jest instead of tape, there should be no new tests added in tape. Or, even better, don't use test runner at all to prevent it's runtime affecting the benchmark result.